src/service/url-replacements-impl.js

```diff
@@ -166,6 +170,20 @@ export class UrlReplacements {
       });
     });
 
+    // Returns assigned variant name for the given experiment.
+    this.set_('VARIANT', experiment => {
+      return this.variants_.then(variants => {
+        user.assert(variants,
```

Comment:
Do you mean:
```
if (error) {
  user.error(...);
  return null;
}
```

If returning null will end up with an empty string, it's identical to user.assert. Because in the replace logic, we catch all errors. See one of my test cases for example.